src/app/dashboard/components/dashboard-visits/dashboard-visits.component.html

```diff
@@ -1 +1,75 @@
-<p>dashboard-visits works!</p>
+<h2>
+  <strong>{{ total }}</strong>
+  {{ 'strings.headingVisits' | translate }}
+</h2>
+
+<ul *ngIf="data" class="visits">
+  <li class="visits-unavailable">
+    <span class="count">{{ data.visits.UNAVAILABLE ?? 0 }}</span>
+    <span class="statistic down" *ngIf="unavailablePercent < 0"
+      ><mat-icon class="material-icons-round icon">keyboard_arrow_down</mat-icon></span
```

Comment:
the code formatting is a bit weird, breaking the span like that
to improve readability, it would look better like this:
<span>
    <icon>
</span>
if you'd like to further improve, you could combine the 3 spans into a single one

Reply:
Resolved.